core/kazoo_xml/src/kz_xml.erl

```diff
@@ -41,3 +42,13 @@ texts_to_binary(Vs, Size) when is_list(Vs), is_integer(Size), Size > 0 ->
 -spec attributes_to_proplist(xml_attribs()) -> kz_proplist().
 attributes_to_proplist(L) ->
     [{K, V} || #xmlAttribute{name=K, value=V} <- L].
+
+-spec filter_empty_text(xml_els() | xml_texts()) -> xml_els() | xml_texts().
```

Comment:
Not exactly; that one is recursive, this one is only the provided list of xml records